bot/cogs/random_cog/random_cog.py

```diff
@@ -133,5 +134,32 @@ async def slotsrolling(input, spinstatus,waittime):
         embed = await slotsrolling(f'{a} | {b} | {c}', f'**{message}**',1.75)
         await msg.edit(embed = embed)
 
+    @commands.command()
+    async def raffle(self, ctx, time: typing.Optional[int] = 5, *, reason):
+        """
+        Raffle command, picks a random winner from users who reacted with :tickets:
+        :param time: optional, timer for the raffle
+        :param reason: required, reason/purpose for the raffle
+        """
+        description = f'Raffle for {reason}\nReact with :tickets: to enter the raffle'
+        embed = discord.Embed(title = 'RAFFLE', color=15158332, description = description)
```

Comment:
we have an enum for colors in the bot, look in consts.py for the color to use. This allows us to have a single source of truth for color themes